documentation/translations/translating.rst

```diff
@@ -108,33 +111,173 @@ details.
      - :github:`GitHub <python/python-docs-uk>`,
        `Transifex <tx_>`_
 
-.. _tx: https://explore.transifex.com/python-doc/python-newest/
-
 
 How to get help
 ===============
 
-Discussions about translations occur on the Python Docs Discord
+If there is already a repository for your language team (there may be links to
+Telegrams/Discords in the ``README``), join and introduce
+yourself. Your fellow translators will be more than happy to help!
+General discussions about translations occur on the Python Docs Discord
 `#translations channel <https://discord.gg/h3qDwgyzga>`_, `translation
-mailing list <translation_ml_>`_, and there's a `Libera.Chat IRC
-<https://libera.chat/>`_ channel, ``#python-doc``.
+mailing list <translation_ml_>`_, and the `translations subsection <_discourse>`_
+of the Python Discourse.
+
+
+Style Guide
+===========
+
+Before translating, you should familiarize yourself with the general
+documentation :doc:`style guide<../style-guide>`. Some translation-specific
+guidelines are explained below.
+
+
+Translate, don't rewrite
+------------------------
+
+Try to stay as close as possible to the original text. Focus on translating its
+meaning rather than rephrasing or rewriting it.
+
+
+Gender neutrality
+-----------------
+
+Many languages use grammatical gender. When possible and natural, prefer
+gender-neutral or inclusive forms. Aim to reflect the inclusive tone of
+the English documentation.
+
+
+Roles and links
+---------------
+
+The Python docs contain many roles (``:role:`target```)
+to other parts of the documentation.
+Leave reStructuredText roles such as ``:func:`print``` or ``:ref:`some-section``` in
+place, even if they contain section titles, because it will break the link.
+If alternate text (``:role:`text <target>``` is provided, it can be translated.
+
+Links (```text <target>`_``) should be handled similarly. If possible, the target
+should be updated to match the language.
+
+.. seealso::
+   :doc:`../markup`
+
+
+Code examples
+-------------
+
+Translate values in code examples, that is string literals, and comments.
+Don't translate keywords or names, including variable, function, class, argument,
+and attribute names. An example of a translated codeblock from the `tutorial <https://docs.python.org/3/tutorial/controlflow.html#keyword-arguments>`_
+is provided below:
+
+.. code-block:: python
+
+   def cheeseshop(kind, *arguments, **keywords):
+       print("-- Czy jest może", kind, "?")
+       print("-- Przykro mi, nie mamy już sera", kind)
+       for arg in arguments:
+           print(arg)
+       print("-" * 40)
+       for kw in keywords:
+           print(kw, ":", keywords[kw])
+
+
+Translation quality
+-------------------
+
+Translators should be proficient in both English and the language they are
+translating to. Translators should aim for a similar level of quality as that
+of the English documentation.
```

Comment:
Not sure if any translation project is using two-phased process of translation: translation/review, but I can imagine that in such process the first-phase translator could be only intermediate in both English and the second language.

I think we should leave the requirements of the languages proficiency up to translation projects. For example, for some intermediate English should be enough, especially at early stages of translation.

I suggest us dropping the first sentence and leaving only the one about the quality.

Reply:
I still would argue that it should be left up to projects decision. With current translation tools, medium English could be enough especially for some low-traffic projects. I think that sometimes not ideal translation can encourage new contributors. I would like us to be inclusive here, inviting everyone with a good will. I think that polishing ones English on such a project should be fine for some teams.

Please note that I'm not suggesting to lower the target quality, rather to allow the process to be more iterative and not limit the contributors because of their education.